internal/app/router.go

```diff
@@ -38,13 +43,16 @@ func (a *Application) NewRouter() http.Handler {
 }
 
 // Получение полного адреса по короткой ссылке
-func (a *Application) findAddr(w http.ResponseWriter, r *http.Request) {
+func (a *Application) handlerFindAddr(w http.ResponseWriter, r *http.Request) {
 
 	short := chi.URLParam(r, "short")
-	addr, ok := a.store.GetAddr(short)
-	if !ok {
-		logger.Warn("not found",
-			"short", short)
+	addr, err := a.store.GetAddr(r.Context(), short)
+	if err != nil {
+		if errors.Is(storage.ErrAddressNotFound, err) {
+			logger.Info(err.Error(), "short", short)
+		} else {
+			logger.Error(err, "short", short)
+		}
 		http.NotFound(w, r)
 		return
```

Comment:
что-то нелогичное. Почему при ошибке notFound?
```suggestion
		if errors.Is(storage.ErrAddressNotFound, err) {
			http.NotFound(w, r)
			return
		}
		
		logger.Error(err, "short", short)
		http. Error(w, r)
		return
```